contrastive_vi/module/contrastive_vi.py

```diff
@@ -578,60 +551,32 @@ def loss(
             inference_outputs["target"],
             generative_outputs["target"],
         )
-        recon_loss = background_losses["recon_loss"] + target_losses["recon_loss"]
-        kl_z = background_losses["kl_z"] + target_losses["kl_z"]
-        kl_s = target_losses["kl_s"]
-        kl_library = background_losses["kl_library"] + target_losses["kl_library"]
-
-        loss = (
-            torch.sum(recon_loss)
-            + torch.sum(kl_z)
-            + torch.sum(kl_s)
-            + torch.sum(kl_library)
+        reconst_loss = background_losses["recon_loss"] + target_losses["recon_loss"]
```

Comment:
This block of changes:

1. Adds our new Wasserstein penalty to the loss.
2. Modifies our loss function to incorporate KL warmup
3. Renames some variables to match the newest version of the scvi-tools example model (apologies for the noise with this).